src/getHumanAge.test.js

```diff
@@ -7,4 +7,64 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array', () => {
+    expect(getHumanAge(1, 1))
+      .toBeInstanceOf(Array);
+  });
+
+  test('should return an array with two elements', () => {
+    expect(getHumanAge(2, 2))
+      .toHaveLength(2);
+  });
+
+  test('should return zeros if age of cat and dog is zero', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should return zeros if age of cat or dog < 15', () => {
+    expect(getHumanAge(5, 11))
+      .toEqual([0, 0]);
+  });
+
```

Comment:
you didn't use here test design technique like boundary values

Reply:
Yes, I realize this is not a boundary values technique. But I use the equivalence class technique and in my opinion, it is appropriate here